### PR TITLE
[INLONG-5889][Sort] Fix MySQL Node JDBC Url For RCE Vulnerability

### LINKED ISSUE #5889: [Bug][Sort] Fix MySQL node JDBC for RCE vulnerability
### What happened

JDBC URL of MySQL was vulnerable to security attacks.

See https://su18.org/post/jdbc-connection-url-attack,

or https://pyn3rd.github.io/2022/06/06/Make-JDBC-Attacks-Brillian-Again-I.

### What you expected to happen

check jdbc url 

### How to reproduce

See https://su18.org/post/jdbc-connection-url-attack,

or https://pyn3rd.github.io/2022/06/06/Make-JDBC-Attacks-Brillian-Again-I.

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Sort

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by EMsnap**
- Fixes #5889 

### Motivation

Fix MySQL Node JDBC Url for RCE vulnerability

### Modifications

Fix MySQL Node JDBC Url for RCE vulnerability

### Verifying this change

run ut 

### Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/util/JdbcUrlUtils.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    private static final Logger LOG = LoggerFactory.getLogger(JdbcUrlUtils.class);
+
+    /**
+     * see https://su18.org/post/jdbc-connection-url-attack/
+     * @param url
+     * @return url after filtering out the invalid property
+     */
+    public static String replaceInvalidUrlProperty(String url) {
+        if (StringUtils.containsIgnoreCase(url, AUTO_DESERIALIZE_TRUE)) {
+            LOG.info("url {} contains invalid property {}, replace it to {}", url,
```
Comment (gong) [inline inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/util/JdbcUrlUtils.java:42]: Maybe, LOG.warn

File: inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/table/JdbcDynamicTableFactory.java [hunk 2]
```diff
@@ -219,6 +221,7 @@ public DynamicTableSource createDynamicTableSource(Context context) {
 
     private JdbcOptions getJdbcOptions(ReadableConfig readableConfig) {
         final String url = readableConfig.get(URL);
+        JdbcUrlUtils.replaceInvalidUrlProperty(url);
```
Comment (gong) [inline inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/table/JdbcDynamicTableFactory.java:224]: `JdbcUrlUtils.replaceInvalidUrlProperty(url);` return value should be used.

File: inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/util/JdbcUrlUtils.java (hunk 1 above)
Comment (EMsnap) [reply inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/util/JdbcUrlUtils.java:42]: will do thx

File: inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/table/JdbcDynamicTableFactory.java (hunk 2 above)
Comment (EMsnap) [reply inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/table/JdbcDynamicTableFactory.java:224]: noted thx

**Review by healchow — APPROVED** [review]

**Review by gong — APPROVED** [review]

**Review by yunqingmoswu — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**